.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx pretty-quick --staged
```

Comment:
What was the reason for running `pretty-quick` with npx here instead of `npm run pretty-quick --staged`?

Reply:
It is the recommended way by `prettier` when using husky
https://prettier.io/docs/en/precommit.html#option-2-pretty-quickhttpsgithubcomazzpretty-quick